### ISSUE TITLE
Support F-order

### ISSUE THREAD

**Issue by sirherrbatka** [marcoheisig/numpy-file-format#10]
Ok, so I think that it wouldn't be that much of a work to do so and it is a little bit annoying to not have this because pandas actually export arrays in the Fortran format. I am willing to implement this. Should I go ahead?

**Comment by marcoheisig** [comment]
Absolutely, that would be wonderful!  Sorry I haven't had time to review your other pull request, but I have an ELS paper to finish right now.  Give me a few more days there.

**Comment by sirherrbatka** [comment]
Yeah, I figured! Good luck with papers!

Marco Heisig pisze:
> Absolutely, that would be wonderful!  Sorry I haven't had time to review
> your other pull request, but I have an ELS paper to finish right now.  Give
> me a few more days there.
-- 
Marek Kochanowicz

**Comment by sirherrbatka** [comment]
So... how are you?

**Comment by marcoheisig** [comment]
@sirherrbatka  Very busy.  For example, I just got sb-simd merged into SBCL.  I am not sure how much free time I'll have until the end of this year.  If you promise not to break things, I could add you as a collaborator for this project so that you could merge stuff yourself.  Would that be ok?

**Comment by sirherrbatka** [comment]
Well, first of all, I am glad that you are alive and well! There is no rush, 
we can get back to it when you have more time available.
